app/models/trace.rb

```diff
@@ -117,7 +117,7 @@ def trace_name
   end
 
   def mime_type
-    filetype = `/usr/bin/file -Lbz #{trace_name}`.chomp
+    filetype = Open3.capture2("/usr/bin/file", "-Lbz", trace_name)[0].chomp
```

Comment:
I tend to prefer `.first` over `[0]` in cases like though I'm not sure there's a really good reason for doing so.

Reply:
Me too! I'm not sure why I overlooked this.